### PR TITLE
Fix titles for AWS Policy packs

### PR DESCRIPTION
**Pull request by madhushreeray30**

### CODE REVIEW

File: policy_packs/aws/ec2/enforce_encryption_by_default_is_enabled_for_ebs_volumes/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ categories: ["security", "storage"]
 primary_category: "security"
 ---
 
-# Enforce Encryption By Default is Enabled for AWS EC2 EBS Volumes
+# Enforce Encryption by Default Is Enabled for AWS EC2 EBS Volumes
```
Comment (cbruno10) [inline policy_packs/aws/ec2/enforce_encryption_by_default_is_enabled_for_ebs_volumes/README.md:6]:
```suggestion
# Enforce Default Encryption Is Enabled for New AWS EBS Volumes
```

File: policy_packs/aws/ec2/enforce_instances_to_not_have_internet_access_via_subnets/README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "networking"
 ---
 
-# Enforce AWS EC2 Instances to Not Have Internet Access Via Subnets
+# Enforce AWS EC2 Instances Do Not Have Internet Access Via Subnets
```
Comment (cbruno10) [inline policy_packs/aws/ec2/enforce_instances_to_not_have_internet_access_via_subnets/README.md:6]:
```suggestion
# Enforce AWS EC2 Instances Do Not Have Internet Access via Subnets
```

File: policy_packs/aws/ec2/enforce_instances_to_use_specific_instance_types/README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ categories: ["cost controls", "security"]
 primary_category: "cost controls"
 ---
 
-# Enforce AWS EC2 Instances to Use Specific Instance Types
+# Enforce AWS EC2 Instances Use Specific Instance Types
```
Comment (cbruno10) [inline policy_packs/aws/ec2/enforce_instances_to_use_specific_instance_types/README.md:6]:
```suggestion
# Enforce AWS EC2 Instances Use Approved Instance Types
```

File: policy_packs/aws/ec2/enforce_tags_on_amis_if_they_are_older_than_14_days/README.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ categories: ["cost controls", "security", "tagging"]
 primary_category: "tagging"
 ---
 
-# Enforce Tags on AMIs if They Are Older Than 14 Days
+# Enforce Tags on AMIs If They Are Older Than 14 Days
```
Comment (cbruno10) [inline policy_packs/aws/ec2/enforce_tags_on_amis_if_they_are_older_than_14_days/README.md:6]:
```suggestion
# Enforce Tags on AMIs if They Are Older Than 14 Days
```

File: policy_packs/aws/iam/deny_all_iam_actions_from_unapproved_networks/README.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "networking"
 ---
 
-# Deny all AWS IAM actions from Unapproved Networks
+# Enforce AWS IAM Actions Only From Approved Networks
```
Comment (cbruno10) [inline policy_packs/aws/iam/deny_all_iam_actions_from_unapproved_networks/README.md:6]:
```suggestion
# Deny all AWS IAM actions from Unapproved Networks
```

File: policy_packs/aws/kms/enforce_rotation_is_enabled_for_keys/README.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ categories: ["data protection", "security"]
 primary_category: "data protection"
 ---
 
-# Enforce Rotation is Enabled on AWS KMS Keys
+# Enforce Rotation Is Enabled on AWS KMS Keys
```
Comment (cbruno10) [inline policy_packs/aws/kms/enforce_rotation_is_enabled_for_keys/README.md:6]:
```suggestion
# Enforce Rotation Is Enabled for AWS KMS Keys
```

File: policy_packs/aws/lambda/enforce_functions_to_use_specific_approved_tags/main.tf [hunk 7]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce AWS Lambda Functions to Use Specific Approved Tags"
+  title       = "Enforce AWS Lambda Functions Use Specific Approved Tags"
```
Comment (cbruno10) [inline policy_packs/aws/lambda/enforce_functions_to_use_specific_approved_tags/main.tf:2]:
```suggestion
  title       = "Enforce AWS Lambda Functions Use Approved Tags"
```

File: policy_packs/aws/s3/enforce_block_public_access_is_enabled_for_accounts/README.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ categories: ["security", "storage"]
 primary_category: "security"
 ---
 
-# Enforce Block Public Access Is Enabled for AWS S3 Accounts
+# Enforce Block Public Access Is Enabled for AWS S3 Buckets
```
Comment (cbruno10) [inline policy_packs/aws/s3/enforce_block_public_access_is_enabled_for_accounts/README.md:6]:
```suggestion
# Enforce Block Public Access Is Enabled for AWS S3 Accounts
```

This pack is for S3 account resources, not buckets.

File: policy_packs/aws/s3/enforce_name_for_buckets_is_dns_compliant/README.md [hunk 9]
```diff
@@ -3,7 +3,7 @@ categories: ["compliance", "storage"]
 primary_category: "compliance"
 ---
 
-# Enforce Name for AWS S3 Buckets Is DNS Compliant
+# Enforce DNS-Compliant Names for AWS S3 Buckets
```
Comment (cbruno10) [inline policy_packs/aws/s3/enforce_name_for_buckets_is_dns_compliant/README.md:6]:
```suggestion
# Enforce AWS S3 Buckets Use DNS-Compliant Names
```

File: policy_packs/aws/vpc/enforce_block_unapproved_network_access_for_security_groups/README.md [hunk 10]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "networking"
 ---
 
-# Enforce Block Unapproved Network Access for AWS VPC Security Groups
+# Enforce Approved Network Access for AWS VPC Security Groups
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_block_unapproved_network_access_for_security_groups/README.md:6]:
```suggestion
# Enforce Unapproved Network Access is Blocked for AWS VPC Security Groups
```

File: policy_packs/aws/vpc/enforce_default_security_groups_to_not_allow_any_access/README.md [hunk 11]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce AWS VPC Default Security Groups to Not Allow Any Access
+# Enforce AWS VPC Default Security Groups to Deny All Access
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_default_security_groups_to_not_allow_any_access/README.md:6]: I'm not sure what this policy pack is doing, is it only looking at default VPC security groups and blocking ingress/egress access?

File: policy_packs/aws/vpc/enforce_default_security_groups_to_not_exist/README.md [hunk 12]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce AWS VPC Default Security Groups to Not Exist
+# Enforce Removal of AWS VPC Default Security Groups
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_default_security_groups_to_not_exist/README.md:6]:
```suggestion
# Enforce AWS VPC Default Security Groups Are Deleted
```

File: policy_packs/aws/vpc/enforce_default_vpcs_to_not_exist/README.md [hunk 13]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce Default AWS VPCs to Not Exist
+# Enforce Removal of Default AWS VPCs
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_default_vpcs_to_not_exist/README.md:6]:
```suggestion
# Enforce Default AWS VPCs Are Deleted
```

File: policy_packs/aws/vpc/enforce_elastic_ips_to_not_be_unassociated/README.md [hunk 14]
```diff
@@ -3,7 +3,7 @@ categories: ["cost controls", "networking", "security"]
 primary_category: "cost controls"
 ---
 
-# Enforce AWS VPC Elastic IPs to Not Be Unassociated
+# Enforce AWS VPC Elastic IPs to Be Associated
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_elastic_ips_to_not_be_unassociated/README.md:6]:
```suggestion
# Enforce AWS VPC Elastic IPs Are Associated
```

File: policy_packs/aws/vpc/enforce_elastic_ips_to_not_exist/README.md [hunk 15]
```diff
@@ -3,7 +3,7 @@ categories: ["cost controls", "networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce AWS VPC Elastic IPs to Not Exist
+# Enforce Removal of AWS VPC Elastic IPs
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_elastic_ips_to_not_exist/README.md:6]:
```suggestion
# Enforce AWS VPC Elastic IPs Are Deleted
```

File: policy_packs/aws/vpc/enforce_enable_flow_logging_to_cloudwatch_log_groups_for_vpcs/README.md [hunk 16]
```diff
@@ -3,7 +3,7 @@ categories: ["logging", "networking", "security"]
 primary_category: "logging"
 ---
 
-# Enforce Enable Flow Logging to CloudWatch Log Groups for AWS VPCs
+# Enforce Flow Logging to CloudWatch Log Groups for AWS VPCs
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_enable_flow_logging_to_cloudwatch_log_groups_for_vpcs/README.md:6]:
```suggestion
# Enforce Flow Logging to CloudWatch Log Groups Is Enabled for AWS VPCs
```

File: policy_packs/aws/vpc/enforce_enable_flow_logging_to_s3_buckets_for_vpcs/README.md [hunk 17]
```diff
@@ -3,7 +3,7 @@ categories: ["logging", "networking", "security"]
 primary_category: "logging"
 ---
 
-# Enforce Enable Flow Logging to S3 Buckets for AWS VPCs
+# Enforce Flow Logging to S3 Buckets for AWS VPCs
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_enable_flow_logging_to_s3_buckets_for_vpcs/README.md:6]:
```suggestion
# Enforce Flow Logging to S3 Buckets Is Enabled for AWS VPCs
```

File: policy_packs/aws/vpc/enforce_internet_gateways_to_not_exist/README.md [hunk 18]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce AWS VPC Internet Gateways to Not Exist
+# Enforce Removal of AWS VPC Internet Gateways
```
Comment (cbruno10) [inline policy_packs/aws/vpc/enforce_internet_gateways_to_not_exist/README.md:6]:
```suggestion
# Enforce AWS VPC Internet Gateways Are Deleted
```

File: policy_packs/aws/iam/enforce_roles_to_have_trusted_access/README.md [hunk 19]
```diff
@@ -3,7 +3,7 @@ categories: ["access management", "security"]
 primary_category: "access management"
 ---
 
-# Enforce AWS IAM Roles to Have Trusted Access
+# Enforce Trusted Access for AWS IAM Roles
```
Comment (cbruno10) [inline policy_packs/aws/iam/enforce_roles_to_have_trusted_access/README.md:6]:
```suggestion
# Enforce AWS IAM Roles Restrict Access
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@madhushreeray30 :
- I didn't make suggestions on all files, typically just the `README.md` title
- Please remember to also update directory names, `akas` in `main.tf`, and any references in README.md as the names should be updated based on the new titles

File: policy_packs/aws/vpc/enforce_default_security_groups_to_not_allow_any_access/README.md (hunk 11 above)
Comment (SumitPopat) [reply policy_packs/aws/vpc/enforce_default_security_groups_to_not_allow_any_access/README.md:6]: Yes, all Ingress and Egress rules are reject for default VPC security groups.

File: policy_packs/aws/vpc/enforce_block_unapproved_network_access_for_security_groups/README.md (hunk 10 above)
Comment (cbruno10) [reply policy_packs/aws/vpc/enforce_block_unapproved_network_access_for_security_groups/README.md:6]: I like using the word `traffic` when talking about VPC security groups

File: policy_packs/aws/ec2/enforce_default_encryption_is_enabled_for_ebs_volumes/main.tf
Comment (SumitPopat) [inline policy_packs/aws/ec2/enforce_default_encryption_is_enabled_for_ebs_volumes/main.tf:1]: I think the directory name and aka should match the title? I.e. `enforce_default_encryption_is_enabled_for_new_ebs_volumes`?

File: policy_packs/aws/vpc/enforce_internet_gateways_are_deleted/README.md [hunk 20]
```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce AWS VPC Internet Gateways to Not Exist
+# [Enforce Removal of AWS VPC Internet Gateways](../enforce_enable_flow_logging_to_s3_buckets_is_enabled_for_vpcs)
```
Comment (SumitPopat) [inline policy_packs/aws/vpc/enforce_internet_gateways_are_deleted/README.md:6]: Please fix this.

File: policy_packs/aws/vpc/enforce_internet_gateways_are_deleted/main.tf [hunk 21]
```diff
@@ -1,5 +1,5 @@
 resource "turbot_policy_pack" "main" {
-  title       = "Enforce AWS VPC Internet Gateways to Not Exist"
+  title       = "policy_packs/aws/vpc/enforce_enable_flow_logging_to_s3_buckets_is_enabled_for_vpcs"
```
Comment (SumitPopat) [inline policy_packs/aws/vpc/enforce_internet_gateways_are_deleted/main.tf:2]: Please fix this title.

**Review by SumitPopat — APPROVED** [review]

**Review by SumitPopat — APPROVED** [review]

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**